src/app/FakeLib/NuGet/NugetHelper.fs

```diff
@@ -91,6 +93,8 @@ let NuGetDefaults() =
       WorkingDir = "./NuGet"
       PublishUrl = null
```

Comment:
There is a [separate](https://github.com/fsharp/FAKE/issues/1241) issue about defaults and I'd prefer to fix it via separate PR. 

> do it like nuget and throw an error

It's a dangerous change, because previous versions of Nuget worked fine without  source specified. If we start to require the source now, it would be a breaking change.